post-microsservice/src/main/kotlin/com/example/postmicrosservice/controller/PostController.kt

```diff
@@ -9,5 +10,18 @@ import org.springframework.web.bind.annotation.RestController
 class PostController(
     private val postService: PostService
 ) {
+    @PostMapping("/create")
+    fun create(post: Post){
+        postService.createPost(post)
+    }
 
+    @PutMapping("/update/{id}")
```

Comment:
I think we don't need to change the URL to create or update. Just add the id. Like we have done in the endpoints PDF
```suggestion
    @PutMapping("/{id}")
```